umbra-js/src/classes/Umbra.ts

```diff
@@ -129,6 +149,7 @@ export class Umbra {
   constructor(readonly provider: EthersProvider, chainConfig: ChainConfig | number) {
     this.chainConfig = parseChainConfig(chainConfig);
     this.umbraContract = new Contract(this.chainConfig.umbraAddress, abi, provider) as UmbraContract;
+    this.batchSendContract = new Contract(batchSendAddress, batchSendAbi, provider) as BatchSendContract;
```

Comment:
Let's add this to the `chainConfig`
```suggestion
    this.batchSendContract = new Contract(this.chainConfig.batchSendAddress, batchSendAbi, provider) as BatchSendContract;
```

Reply:
Added in b5ed2dde8d6d175dc42203ca8d5a04106ecc5777